src/main/kotlin/com/project/twiliospring/repository/impl/F2ARepositoryImpl.kt

```diff
@@ -0,0 +1,28 @@
+package com.project.twiliospring.repository.impl
+
+import com.project.twiliospring.domain.F2ARecord
+import com.project.twiliospring.repository.F2ARepository
+import org.springframework.stereotype.Component
+import java.time.LocalDateTime
+
+@Component
+class F2ARepositoryImpl : F2ARepository {
+    val storage = LinkedHashMap<String, F2ARecord>()
```

Comment:
applicable only in case if we run only one instance of application. For applications in k8s that have many pods this solution won't work

Reply:
Changed to store records in Redis